lib/inventory_sync/async/inventory_scheduled_sync.rb

```diff
@@ -0,0 +1,19 @@
+module InventorySync
+  module Async
+    class InventoryScheduledSync < ::Actions::EntryAction
+      include ::Actions::RecurringAction
+
+      def plan
+        concurrence do
+          Organization.unscoped.each do |org|
+            plan_org_sync(org)
```

Comment:
Is having a method for it really necessary?

Reply:
I am using those methods for testing. It's much easier to mock a method on this class than the actual call to foreman tasks.